README.md

```diff
@@ -1,50 +1,49 @@
 # OpenEBS Monitoring add-on
 
-This repository contains monitoring-related (like Prometheus, grafana, etc,) artifacts like helm charts/ YAMLs. The goal of this repository is to provide an easy to setup monitoring stack for OpenEBS.
+A set of Grafana dashboards and Prometheus alerts for OpenEBS, packaged as helm chart.
 
-This repository will aggregate all the monitoring related artifacts that are currently spread across multiple repositories like:
-
-- https://github.com/openebs/openebs/tree/master/k8s
-- https://github.com/openebs/charts/tree/gh-pages/grafana-charts
+You have the following options for deploying OpenEBS monitoring dashboards
+- Install [OpenEBS Monitoring Add-on](#install) that includes Prometheus Operator and OpenEBS Mixins.
+- Install Only [OpenEBS Mixins](./docs/guide.md).
 
 ## Status
 
 Pre-alpha. Under active development.
 
-## Usage
+## Install
 
-[Helm](https://helm.sh) must be installed to use the charts.
-Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
+### Install using Helm Chart
 
-Once Helm is set up properly, add the repo as follows:
+- Step 1: [Helm](https://helm.sh) must be installed to use the charts. Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
 
-```console
-helm repo add openebs-monitoring https://openebs.github.io/monitoring/
-```
+- Step 2: Add OpenEBS Monitoring chart to Helm using the following command:
+  ```console
+  helm repo add openebs-monitoring https://openebs.github.io/monitoring/
+  ```
 
-You can then run `helm search repo openebs-monitoring` to see the charts.
+  You can then run `helm search repo openebs-monitoring` to see the charts.
 
-#### Install Chart
+- Step 3: Install OpenEBS Monitoring add-on. 
```

Comment:
If the K8s distribution is MicroK8s, it is necessary to enable the dns addon before installing the monitoring charts since Grafana needs ClusterDNS. One can do so with the command:
microk8s.enable dns